src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -47,6 +47,23 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
   ui_->unwindingMethodComboBox->addItem("DWARF", static_cast<int>(CaptureOptions::kDwarf));
   ui_->unwindingMethodComboBox->addItem("Frame pointers",
                                         static_cast<int>(CaptureOptions::kFramePointers));
+
+  QObject::connect(ui_->unwindingMethodComboBox, &QComboBox::currentTextChanged,
+                   ui_->unwindingMethodWidget, [this](const QString& /*unused*/) {
+                     auto unwinding_method = static_cast<UnwindingMethod>(
+                         ui_->unwindingMethodComboBox->currentData().toInt());
+                     if (unwinding_method == CaptureOptions::kFramePointers) {
+                       ui_->unwindingMethodWidget->setStyleSheet("QWidget { color:#ff8000; }");
```

Comment:
As an instinct, I would consider making only the text in the combo box ("Frame pointers") orange. I feel like the current solution stands out too much.